src/modules/sync/sync/collections.ts

```diff
@@ -1,28 +1,45 @@
 import { CollectionModels, ManagementClient } from "@kontent-ai/management-sdk";
 
+import { logInfo, LogOptions } from "../../../log.js";
 import { omit } from "../../../utils/object.js";
 import { DiffModel } from "../types/diffModel.js";
 import { getTargetCodename, PatchOperation } from "../types/patchOperation.js";
 
-export const syncAddAndReplaceCollections = (client: ManagementClient, collections: DiffModel["collections"]) => {
+export const syncAddAndReplaceCollections = (
+  client: ManagementClient,
+  collections: DiffModel["collections"],
+  logOptions: LogOptions,
+) => {
   if (!collections.length) {
+    logInfo(logOptions, "standard", "No collections to update");
     return Promise.resolve();
   }
 
+  logInfo(logOptions, "standard", "Updating collections");
```

Comment:
To avoid confusion that there is no "Adding collections", I would instead use "Adding and updating collections". Similarly for the "No..." version.